brainstormforce/ultimate-addons-for-gutenberg
# Price List - Issue of block validation when image align set to middle

### Description

- Issue of block validation when image align set to middle
- Price list content width vary issue - https://share.getcloudapp.com/2Nu0n8e2
- Trello - https://trello.com/c/dCguVRVC/181-price-list-width-vary-according-to-content-issue

### Types of changes




-  Bug fix (non-breaking change which fixes an issue) 

### How has this been tested?

- Drag and drop Price list in master.
- Set image alignment vertically middle update page and refresh the editor (issue occured).
- switch to price-list-img-issue branch and update page and refresh page issue solved.
- Same create price list in master with Set 1 column & enter a long description check front-end (Issue occured).
- Switch to branch and refresh and check again ( Issue solved ).

### Checklist:
